### DISCUSSION TITLE
My score is still 82 in pagespeed though individual metrics are in green

### DISCUSSION THREAD

**Discussion by skbe1681** [GoogleChrome/lighthouse#13584] [Q&A]
url : https://olamnuts.com

Pagespeed overall score - 82

Today individual metrics was
FCP - .9 s (green)
TTI - .9 s (green)
SI - 1.3 s (green)
TBT - 0ms (green)
LCP - 2.9 s (red)
CLS - .016 (green)

Most of the time for last 22 days, we had green FCP, TTI, SI, TBT and CLS but score was in the range of 82 to 86 %.

How does the individual metrics affects score in page speed ? Please share some more insights on the calculation.

**Comment by brendankenny** [comment]
<img width="399" alt="screenshot of the Lighthouse score gauge with links underneath" src="https://user-images.githubusercontent.com/316891/150400922-da0403c0-98c2-4d64-8056-43693479e1b5.png">

In the Lighthouse report, underneath the score gauge are two links that might help explain things.

- an explanation of how Lighthouse scoring is done: [Lighthouse performance scoring](https://web.dev/performance-scoring/)
- an [interactive score calculator](https://googlechrome.github.io/lighthouse/scorecalc/) that shows how the metrics are weighted and allows adjusting metric values and seeing how the overall performance score changes

The second link is pre-populated with the metric values seen if clicked through directly from a report.
